Source/utils/sdl2_to_1_2_backports.cpp

```diff
@@ -792,6 +795,8 @@ char *SDL_GetBasePath()
 		retval = readSymLink("/proc/curproc/exe");
 #elif defined(__QNXNTO__)
 		retval = SDL_LoadFile("/proc/self/exefile", NULL);
+#elif defined(__amigaos__)
+		//avoid compiling readSymLink	

```

Comment:
Since the previous id-def block returns for both 3DS and Amiga we should probably add an else to it that wraps the rest of the code instead of this.

Reply:
True.